src/freezeImage.js

```diff
@@ -0,0 +1,19 @@
+const freezeImage = (selector) => {
+  const anchor = document.querySelector(selector);
+  const src = anchor.src;
+  const className = anchor.className;
+  const canvas = document.createElement('canvas');
+  canvas.className = className;
+  const context = canvas.getContext('2d');
+  anchor.parentNode.replaceChild(canvas, anchor);
+
+  const image = new Image();
```

Comment:
We are trying to redraw the new image and we replace the image tag. Apparently, this is how the drawImage works